index.html

```diff
@@ -28,20 +28,57 @@
     <main>
       <div class="main-section-grid">
         <div class="left-menu left-main-grid">
-          <ul>
-            <div class="home-main">Home</div>
-            <div class="explore">Explore</div>
-            <div class="notifications">Notifications</div>
-            <div class="messages">Messages</div>
-            <div class="bookmarks">Bookmarks</div>
-            <div class="lists">Lists</div>
-            <div class="profile">Profile</div>
-            <div class="more">More</div>
-          </ul>
-
-          <div class="tweet"></div>
-
-          <div class="profile-info"></div>
+          <!-- <ul class="left-menu-links"> -->
+          <div class="home-main">
+            <div class="home-image">Image</div>
+            <div class="home-button">Home</div>
+          </div>
+
+          <div class="explore">
+            <div class="explore-image">image</div>
+            <div class="explore-button">explore</div>
+          </div>
+
+          <div class="notifications">
+            Notifications
+            <div class="notifications-image"></div>
+            <div class="notifications-button"></div>
+          </div>
```

Comment:
There are semantically more accurate tags for navigation, which means people using assisted technology like screen readers can tab through your navigation menu. Something like below? If you have <div> around your image and text within the <a>, they both serve as links

`<nav class="s">
            <ul class="">
                <a href="#">
                    <div class="frame">
                        <img src="/images/assets/icons/bookmark-32.png" alt="home icon" />
                    </div>
                    <div class="text"> Home</div>
                </a>
            </ul>
        </nav>`